### ISSUE TITLE
Number of classes

### ISSUE THREAD

**Issue by RaptorMai** [lifelong-robotic-vision/OpenLORIS-Object#3]
According to the paper, the dataset has 121 instances including 40 categories. But when I run the code of this repo, I found that there are 68 classes in each of the clutter, illumination, occlusion and pixel datasets. 

My questions are:
- what do instance and category mean? 
- What's are the total number of classes in the dataset?

This is the snippet I used to check the number of classes. 
```
with open(args.factor + '.pk', 'rb') as f:
    ((train_datasets, test_datasets), config, classes_per_task) = pickle.load(f)
for i in range(len(train_datasets)):
    print(set(train_datasets[i].labels))
```

**Comment by sheqi** [comment]
hi,

Thanks for your interest. Yeah, we have 2 versions for openloris-object, one is 68 (used for iros 2019 competition); another is 121 (used in the icra paper).

For your question: 

- what do instance and category mean?
The category means the object class, for example, we have  ``"bottle" as the class label; the instance means that the bottle may be red/blue/black bottle, we indicate as "bottle 1/2/3" as the label.``

- What's are the total number of classes in the dataset?
The total number is 121. 

BTW, a quick question, have you seen instruction here ?
https://docs.google.com/document/d/1KlgjTIsMD5QRjmJhLxK4tSHIr0wo9U6XI5PuF8JDJCo/edit?usp=sharing

@ffeng1996 please also help follow the issue.

**Comment by ffeng1996** [comment]
Hi,

Thanks for your interests. 

Yes, for this code base, we only used 68 classes. I will update the data loader to fit the 121 classes by this week. Please follow this link to download the dataset: [download link](https://docs.google.com/document/d/1KlgjTIsMD5QRjmJhLxK4tSHIr0wo9U6XI5PuF8JDJCo/edit?usp=sharing).

**Comment by ffeng1996** [comment]
Hi,

I have checked that it have already included 121 objects. 

Could you please check whether you used [this version](https://drive.google.com/file/d/11jgiPB2Z9WRI3bW6VSN8fJZgwFl5mLsF/view?usp=sharing)? 

Please feel free to ask me if you encounter any difficulties.
